test/rewrites.lean

```diff
@@ -43,30 +44,9 @@ info: Try this: rw [@mul_left_eq_self]
 example [Group G] (h : G) : 1 * h = h := by
   rw?!
 
+#guard_msgs(drop info) in
 example [Group G] (g h : G) : g * g⁻¹ * h = h := by
   rw? -- the right answer is not the first solution, so we can't use rw?!
-  /- Prints:
-  rw [@Semigroup.mul_assoc]
-  -- g * (g⁻¹ * h) = h
-  rw [@mul_assoc]
-  -- g * (g⁻¹ * h) = h
-  rw [@mul_left_eq_self]
-  -- g * g⁻¹ = 1
-  rw [@mul_inv_self]
-  -- 1 * h = h
-  rw [@mul_right_inv]
-  -- 1 * h = h
-  rw [← @division_def]
-  -- g / g * h = h
-  rw [← @div_eq_mul_inv]
-  -- g / g * h = h
-  rw [← @DivInvMonoid.div_eq_mul_inv]
-  -- g / g * h = h
-  rw [@inv_eq_one_div]
-  -- g * (1 / g) * h = h
-  rw [← @mulRightEmbedding_apply]
-  -- ↑(mulRightEmbedding h) (g * g⁻¹) = h
-  -/
   rw [mul_inv_self]
```

Comment:
@semorrison Could you review this `test/rewrites.lean` file? I'm not sure exactly what you want to test here.

I'm happy with the changes in the rest of the files.

Reply:
This test has been deleted recently, so this query becomes obsolete after merging master.